integration-testing/src/test/java/org/triplea/server/remote/actions/RemoteActionsWebsocketIntegrationTest.java

```diff
@@ -0,0 +1,90 @@
+package org.triplea.server.remote.actions;
```

Comment:
This is an end-to-end test to verify the new 'remote-actions-websocket'. We spin up a server, connect with a websocket, use HTTP client to do a ban and shutdown request, and we then verify the websocket listener gets a message.

I am a bit concerned about potential non-determinism that could result from this test, we are exercising a *lot* of code through a real server and real database.

Reply:
Removing this test, it's not fully deterministic. I also suspect this is falling too far into integration test territory, not clear how this test could independently fail.